ProcessLib/LiquidFlow/LiquidFlowProcess.cpp

```diff
@@ -0,0 +1,124 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   LiquidFlowProcess.cpp
+ *
+ * Created on August 19, 2016, 1:38 PM
+ */
+
+#include "LiquidFlowProcess.h"
+
+#include <cassert>
+
+#include "ProcessLib/Utils/CreateLocalAssemblers.h"
+#include "LiquidFlowLocalAssembler.h"
+
+#include "MaterialLib/PorousMedium/Porosity/Porosity.h"
+#include "MaterialLib/PorousMedium/Storage/Storage.h"
+
+namespace ProcessLib
+{
+namespace LiquidFlow
+{
+LiquidFlowProcess::LiquidFlowProcess(
+    MeshLib::Mesh& mesh,
+    std::unique_ptr<AbstractJacobianAssembler>&& jacobian_assembler,
+    std::vector<std::unique_ptr<ParameterBase>> const& parameters,
+    unsigned const integration_order,
+    std::vector<std::reference_wrapper<ProcessVariable>>&& process_variables,
+    SecondaryVariableCollection&& secondary_variables,
+    NumLib::NamedFunctionCaller&& named_function_caller,
+    const bool compute_gravitational_term,
```

Comment:
Changed the order (also in LiquidFlowLocalAssembler). See the amended commit: 5421473.